README.md

```diff
@@ -54,6 +54,9 @@ repos:
     url: http://pkgbuild.com/~anatolik/quarry/x86_64
   sublime:
     url: https://download.sublimetext.com/arch/stable/x86_64
+  archlinux-reflector:
+    mirrorlist: /etc/pacman.d/reflector_mirrorlist # Be careful! Check that pacoloco URL is NOT included in that file!
+#   mirrorlist_refresh_cron: 0 0 3 * * * * # Optional, it is being refreshed before prefetching. Use this setting if you do not use prefetching or you want to update it more often
```

Comment:
Do we really need this mirrorlist cron option? Is it possible to do the reloading automatic without requiring a lot of resources?

I feel that following algorithm would be better: when a user makes a request to some repo, pacoloco checks mirrorlist file modification time, if it shows this file was modified since the last time it is used then the file get reloaded.

Reply:
Fully agree, I didn't like that solution me too